tracer/test/Datadog.Trace.IntegrationTests/SendTracesToAgent.cs

```diff
@@ -1,123 +0,0 @@
-// <copyright file="SendTracesToAgent.cs" company="Datadog">
-// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
-// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
-// </copyright>
-
-using System;
-using System.Linq;
-using System.Net;
-using System.Threading.Tasks;
-using Datadog.Trace.Agent;
-using Datadog.Trace.Configuration;
-using Datadog.Trace.TestHelpers;
-using Datadog.Trace.TestHelpers.HttpMessageHandlers;
-using Xunit;
-
-namespace Datadog.Trace.IntegrationTests
-{
-    public class SendTracesToAgent
-    {
-        private readonly Tracer _tracer;
-        private readonly RecordHttpHandler _httpRecorder;
-
-        public SendTracesToAgent()
-        {
-            var settings = new TracerSettings();
-
-            var endpoint = new Uri("http://localhost:8126");
-            _httpRecorder = new RecordHttpHandler();
-            var api = new Api(endpoint, apiRequestFactory: null, statsd: null);
-            var agentWriter = new AgentWriter(api, statsd: null);
-
-            _tracer = new Tracer(settings, agentWriter, sampler: null, scopeManager: null, statsd: null);
-        }
-
-        [Fact(Skip = "Run manually")]
-        public async Task MinimalSpan()
-        {
-            var scope = _tracer.StartActive("Operation");
-            scope.Dispose();
-
-            // Check that the HTTP calls went as expected
-            await _httpRecorder.WaitForCompletion(1);
-            Assert.Single(_httpRecorder.Requests);
-            Assert.Single(_httpRecorder.Responses);
-            Assert.All(_httpRecorder.Responses, (x) => Assert.Equal(HttpStatusCode.OK, x.StatusCode));
-
-            var trace = _httpRecorder.Traces.Single();
-            MsgPackHelpers.AssertSpanEqual(scope.Span, trace.Single());
-        }
-
-        [Fact(Skip = "Run manually")]
-        public async Task CustomServiceName()
-        {
-            const string ServiceName = "MyService";
-
-            var scope = _tracer.StartActive("Operation", serviceName: ServiceName);
-            scope.Span.ResourceName = "This is a resource";
-            scope.Dispose();
-
-            // Check that the HTTP calls went as expected
-            await _httpRecorder.WaitForCompletion(1);
-            Assert.Single(_httpRecorder.Requests);
-            Assert.Single(_httpRecorder.Responses);
-            Assert.All(_httpRecorder.Responses, (x) => Assert.Equal(HttpStatusCode.OK, x.StatusCode));
-
-            var trace = _httpRecorder.Traces.Single();
-            MsgPackHelpers.AssertSpanEqual(scope.Span, trace.Single());
-        }
-
-        [Fact(Skip = "Run manually")]
-        public async Task Utf8Everywhere()
-        {
-            var scope = _tracer.StartActive("Aᛗᚪᚾᚾᚪ", serviceName: "На берегу пустынных волн");
-            scope.Span.ResourceName = "η γλώσσα μου έδωσαν ελληνική";
-            scope.Span.SetTag("யாமறிந்த", "ნუთუ კვლა");
```

Comment:
Once upon a time this test exposed some UTF-8 issues in the server-side code. Bye!

Reply:
Interesting, well, we are still testing our _Mock_ tracer agent with this 😄